packages/features/ee/workflows/lib/aphanumericSenderIdSupport.ts

```diff
@@ -0,0 +1,75 @@
+export const noAlphanumericSenderIdSupport = [
```

Comment:
List of country codes that don't support alphanumeric sender IDs: 
[Supported Countries](https://support.twilio.com/hc/en-us/articles/223133767-International-support-for-Alphanumeric-Sender-ID)
Country codes from here: [Country Codes](https://countrycode.org/)

Reply:
do we throw an error if the country is not supported?